tutorials/clustering.md

```diff
@@ -22,15 +25,92 @@ need a common superuser role on each postgres instance. This tutorial
+### Update your Postgres configuration
+
+First of all, if you haven't yet run the `timescaledb-tune` utility, it's
+highly recommended that you do so before starting. Do this for all of your
+nodes.
+```
+sudo timescaledb-tune
+```
+
+Also, while you're visiting each of your nodes, make sure you've enabled
+prepared transactions on them, as these are essential for our distributed
+transactions. By default these are disabled, so add the following line to
+your `postgresql.conf`.
+```
+max_prepared_transactions = 150
+```
+
+### Set password encryption
+
+The next thing we'll be doing is setting up the network communication between
+our nodes and adding authentication. For more details and option for this step,
+please visit the [data node authentication][data-node-authentication] page.
+
+In this example we'll be adding `scram-sha-256` password authentication to your
+nodes and adding a password entry for the postgres role as well as a new user
+role we'll be creating later on. The first step of this is to enable password
+encryption on all the nodes by adding the following to `postgresql.conf`:
+
+```
+password_encryption = 'scram-sha-256'
+```
+
+>:WARNING: This setting is not automatically applied to any existing passwords.
+Any existing roles will have to update to new passwords to take advantage of
+the encryption.
+
+After making this change, restart the postgres service so the changes made so
+far can take effect.
+
+### Set up the access node
+
+The next step is to enable `access.example.com` to send the passwords for the
+various users of the system. Create a new file `passfile` in the Postgres data
+directory, and add the following lines:
+```
+*:*:*:postgres:postgres
+*:*:*:testuser:testpass
+```
+
+This will result in the access node always using password `postgres` for role
+`postgres` and `testpass` for role `testuser` whenever connecting to a data
+node. Make sure that this file is readable by the postgres user.
+
+### Set up the data nodes
+
+Next we will set up the data node to accept incoming connections from the
+access node for the roles we expect. The first thing you need to do here is
+to make sure that Postgres will listen for incoming connections. Add the
+following to `postgresql.conf`:
+```
+listen_addresses = '*'
+```
+
+Next, add the roles you want to allow to `pg_hba.conf`:
+```
+host    all    postgres    access.example.com    scram-sha-256
+host    all    testuser    access.example.com    scram-sha-256
+```
+
+This allows connections to the `postgres` and `testuser` roles, but only from
+the access node, and only using `scram-sha-256` authentication.
+
+Finally, we need to make sure the `postgres` role's password matches the
+password we set in the `passfile` on the access node. Run the following
+command in psql.
```

Comment:
make clear that `psql` must connect to the data node. In fact, probably better to say "run the following command on all data nodes."